test_scripts/API/AddSubMenu/020_ATF_AddSubMenu_Positive_SubMenuIcon_DYNAMICType.lua

```diff
@@ -81,7 +78,7 @@ function Test:AddSubMenu_SubMenuIconDYNAMIC()
     subMenuIcon =
     {
       imageType = "DYNAMIC",
-      value = storagePath .. "menuIcon.jpg"
+      value = "menuIcon.jpg"
```

Comment:
@dboltovskyi, for "DYNAMIC" imageType, expected value should have full SDL path (for CI storagePath .. "menuIcon.jpg" , locally SDL returns "./storage...", please return verification: "value = storagePath .. "menuIcon.jpg""

For info: when imageType = "STATIC" SDL uses binary and the check is value = "menuIcon.png"

Please note that for now there is SDL defect regarding imageType = "DYNAMIC" that is observed for PerformAudioPassThru: "[GENIVI] SDL sends incorrect path for Dynamic image with audioPassThruIcon"